float32 reward_value
bool episode_done
